ipu-plugin/pkg/ipuplugin/deviceplugin.go

```diff
@@ -8,27 +8,30 @@ import (
 	"slices"
 	"strings"
 
+	"github.com/intel/ipu-opi-plugins/ipu-plugin/pkg/types"
 	pb "github.com/openshift/dpu-operator/dpu-api/gen"
 	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
 )
 
 type DevicePluginService struct {
 	pb.UnimplementedDeviceServiceServer
+	mode string
```

Comment:
What provides this mode right now?

Reply:
It is the LifeCycleService?

Reply:
https://github.com/intel/ipu-opi-plugins/blob/main/ipu-plugin/ipuplugin/cmd/rootcmd.go#L167

Reply:
Thanks!